### DISCUSSION TITLE
Two Monitors per segment?

### DISCUSSION THREAD

**Discussion by DeNohvo** [awawa-dev/HyperHDR#929] [Q&A]
I have one WLED controller but have divided the strip between two monitors, how can i get HyperHDR to screen capture both monitors and output the correct effect on each segment?

**Comment by awawa-dev** [comment]
https://github.com/awawa-dev/HyperHDR/discussions/885

**Reply by DeNohvo** [reply]
We can create different LED hardware in HyperHDR but both will use a single monitor.

Even if I setup the monitors with different LED controllers you still cannot assign individual controllers to separate monitors.

Looks like Hyperion has this option using QT, it would be nice if this could be added?

**Reply by awawa-dev** [reply]
DirectX 11 grabber doesn't provide such option. It's necessary to create a separate grabber instance for the second monitor (so the GFX resource goes up * 2), but such model with multiple grabbers is currently unsupported in HyperHDR. Maybe in future something will change. Current solution is: first monitor capturing using HyperHDR DX11 grabber + Hyperion Screen grabber and sending captured second monitor via flatbuffers to HyperHDR.

**Comment by awawa-dev** [comment]
Hi @JonSnow88 
Added support for multiple monitors in Windows. Feel free to test.

PR:
https://github.com/awawa-dev/HyperHDR/pull/966
Installers:
https://github.com/awawa-dev/HyperHDR/actions/runs/11594672924
